example/shared_bindings/ffigen_configs/a_shared_base.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2022, the Dart project authors. Please see the AUTHORS file
+# for details. All rights reserved. Use of this source code is governed by a
+# BSD-style license that can be found in the LICENSE file.
+
+name: NativeLibraryASharedB
+description: Bindings to `headers/a.h` with shared definitions from `headers/base.h`.
+output: '../generated/a_shared_b_gen.dart'
+headers:
+  entry-points:
+    - '../headers/a.h'
+library-imports:
+  base_h: '../generated/base_gen.dart'
+symbol-file-map:
+  base_h: '../generated/base_symbols.yaml'
```

Comment:
Sounds good.

Reply:
Implemented this.

I also went ahead to remove the `symbol-file-map` config completely and merging it with `library-imports`.
So the config now looks like this - 
```yaml
library-imports:
  old_format: 'old/format/still/works.dart'
  base_h:
    import-path: '../generated/base_gen.dart'
    symbol-file: '../generated/base_symbols.yaml'
```

I.m.o we should keep `import-path` mandatory, since to make it optional, we need to take import-path as input from the user while generating symbols.yaml file as well.

Reply:
I guess that's true.

The problem I was trying to avoid solving was of invalid import-path for shared bindings (in a single project) if a user specifed a relative path.
We can either only allow package uri path for import-path in symbols.dart or allow the end user to override the import-path instead.
wdyt?

Reply:
> The problem I was trying to avoid solving was of invalid import-path for shared bindings (in a single project) if a user specifed a relative path.

That could indeed be a problem.
If the user does intend to make it available to other packages, their first release will reveal the issue.
On the other hand, if the user is not intending to make the bindings available to other packages, it could be fine to have a relative path (though only if you only import it from specific locations in your project, so probably still not what you want).

> We can either only allow package uri path for import-path in symbols.dart or allow the end user to override the import-path instead.

Maybe we should just add a warning when generating.